Some minor edits to existing entry plus the remainder of the research added (#194)
DFE-Digital/bat-design-history: app/posts/publish-teacher-training-courses/2019-11-01-accredited-bodies-research-round-3.md

### UCAS Weblink

The UCAS Weblink is used by both accredited bodies with SRS and accredited bodies without SRS to manage applications from candidates. It lists all the providers and the applications that have come through for those partners and what status the applications are though it has it's limitations. ABs will only see the details about what is being applied to, and not all the courses they are accredited bodies for.

> For us to go in and check the data for our partners, we’d have to go in the same way as an applicant would and that really isn’t an efficient way of working.

### Issues with the application process

> The processing of importing applications from the SRS is generally automatic and takes place daily, or at specific times configured by the accredited body. The UCAS Weblink connects UCAS with the providers SRS.

Reviewer: ```suggestion
The process of importing applications from the SRS is generally automatic and takes place daily, or at specific times configured by the accredited body. The UCAS Weblink connects UCAS with the providers SRS.
```

{{ appFigure({
  image: {
    file: "application-process.png"
  }
}) }}

> If a course does not yet exist in the SRS or if all course data does not map to a providers records, the application does not get assigned to a course and instead, is effectively blocked and placed in an import error list. This is sometimes referred to as the staging area within the SRS.

Reviewer: ```suggestion
If a course does not yet exist in the SRS or if all course data does not map to a provider’s records, the application does not get assigned to a course and instead, is effectively blocked and placed in an import error list. This is sometimes referred to as the staging area within the SRS.
```

### Access roles and permissions

Multiple participants mentioned the need for a central point for information that can be accessed by multiple team members with varying administrative permissions.

Many providers currently use spreadsheets to internally track and manage applicant and course information. The spreadsheets get circulated internally via email and easily become out of date.

> There is a need to be able to access the information from one data source but with different administrative abilities, not everybody should have admin rights.

One provider expressed the need for a verification/approval step before courses gets published.

> We want to be able to verify course changes and new courses before they go onto Find.

Several providers indicated they need more levels of permission in Publish.

> We wouldn't want publishing functionality for all users - people clicking on things they shouldn’t. If we could have a standard account as read only, and an admin account with the privilege to publish and unpublish, that would be great.

> ### Courses and an accredited body prototype feedback

Reviewer: ```suggestion
### Courses as an accredited body prototype feedback
```

Participants suggested a need for more information on the Courses and accredited screens. This included:

* vacancy data per location when a course had multiple sites
* approved allocation numbers

Access and permissions were also mentioned.

> We’d like tutors to be able to see (but not be able to edit) this information.

> 

Reviewer: Delete whitespace
